docs/schema.md

```diff
@@ -15,11 +15,91 @@ let schema = [
 <SignUpFormParams schema>
 ```
 
-### Required
-For required changes. For now Required checks if the value is `""` to throw the error.
+### NoValidation
+Use for optional fields. NoValidation accepts any or no value
 ```reason
 let schema = [
```

Comment:
this doesn't exist anymore right

Reply:
Exactly, this existed in Reform_Validation no more in ReFormNext